ratatui/src/widgets/list/rendering.rs

```diff
@@ -69,7 +68,7 @@ impl StatefulWidgetRef for List<'_> {
         state.offset = first_visible_index;
 
         // Get our set highlighted symbol (if one was set)
-        let highlight_symbol = self.highlight_symbol.unwrap_or("");
+        let highlight_symbol = self.highlight_symbol.clone().unwrap_or_default();
```

Comment:
Does as_ref() work here instead of clone?

Reply:
Updated in https://github.com/ratatui/ratatui/pull/1467/commits/095bf84245697a19568bbce422c5ea9592305fd1